docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:2.7.13-slim
+
+WORKDIR /app/
+RUN groupadd --gid 10001 app && useradd -g app --uid 10001 --shell /usr/sbin/nologin app
+
+# Install OS-level things
+COPY ./docker/set_up_ubuntu.sh /tmp/
+RUN DEBIAN_FRONTEND=noninteractive /tmp/set_up_ubuntu.sh
+
+# Install breakpad bits
+COPY ./scripts/build-breakpad.sh /tmp
+COPY ./minidump-stackwalk/ /tmp/minidump-stackwalk/
+COPY ./docker/set_up_stackwalk.sh /tmp
```

Comment:
I should merge `scripts/build-breakpad..sh` into `docker/set_up_stackwalk.sh` to continue to reduce the complexity.